tests/e2e/env/shared.sh

```diff
@@ -25,7 +25,7 @@ redirect_output() {
     fi
 }
 
-# --user xfs forces the wordpress:cli container to use a user with the same ID as the main wordpress container. See:
+# --user 33:33 forces the wordpress:cli container to use a user with the same ID as the main wordpress container. See:
```

Comment:
Is this comment update needed? Given we didn't update the E2E script in this PR.

Reply:
The E2E script already used `33:33` so I just fixed the comment to reflect that.